vendor/modules.txt

```diff
@@ -99,6 +99,7 @@ github.com/gophercloud/gophercloud/openstack/compute/v2/flavors
 github.com/gophercloud/gophercloud/openstack/compute/v2/servers
 github.com/gophercloud/gophercloud/openstack/identity/v2/tenants
 github.com/gophercloud/gophercloud/openstack/identity/v2/tokens
+github.com/gophercloud/gophercloud/openstack/identity/v3/applicationcredentials
```

Comment:
how come that this has changed without changing go.mod?

Reply:
It was already in go.mod after the new authentication types PR.

Reply:
Ack